src/gt4py/backend/gtc_backend/defir_to_gtir.py

```diff
@@ -132,13 +133,28 @@ def visit_StencilDefinition(self, node: StencilDefinition) -> gtir.Stencil:
         field_params = {f.name: self.visit(f) for f in node.api_fields}
         scalar_params = {p.name: self.visit(p) for p in node.parameters}
         vertical_loops = [self.visit(c) for c in node.computations if c.body.stmts]
+        if node.externals is not None:
+            externals = {name: str(value) for name, value in node.externals.items()}
+        else:
+            externals = {}
         return gtir.Stencil(
             name=node.name,
+            api_signature=[
+                gtir.Argument(
+                    name=f.name,
+                    is_keyword=f.is_keyword,
+                    default=str(f.default) if not isinstance(f.default, type(Empty)) else "",
```

Comment:
Isn't `type(Empty) == type`? Not sure what the motive is here or is it a type?

Reply:
I'll try removing it!